pixeltable/dataframe.py

```diff
@@ -170,6 +208,9 @@ def show(self, n: int = 20) -> DataFrameResultSet:
 
     def head(self, n: int = 20) -> DataFrameResultSet:
         return self.show(n)
+    
+    def _column_names(self):
```

Comment:
Given that you refer to aliases elsewhere, best to use that term here as well for consistency.